src/client/discord.h

```diff
@@ -0,0 +1,57 @@
+/*
+Minetest
+Copyright (C) 2018 Dumbeldor, Vincent Glize <vincent.glize@live.fr>
+
+This program is free software; you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation; either version 2.1 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License along
+with this program; if not, write to the Free Software Foundation, Inc.,
+51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+
+#pragma once
+#include <iostream>
+#include <memory>
+
+struct DataRichPresence
+{
+	std::string state = "";
+	std::string details = "";
+	uint64_t startTimestamp;
+	std::string largeImageKey = "default";
```

Comment:
Use underscores instead of camelCase. i.e. `large_image_key` and so on for the other variable names.